packages/salesforcedx-vscode-apex/test/jest/index.test.ts

```diff
@@ -62,3 +69,23 @@ describe('indexDoneHandler', () => {
     expect(apexLSPStatusBarItemMock).toHaveBeenCalledTimes(1);
   });
 });
+
+class MockApexLanguageClient {
+  constructor() { }
+  start() { }
+  stop() { }
+}
```

Comment:
I'm not sure why this is necessary? 

Would 
```
stopSpy = jest.fn();
jest.spyOn(languageClientUtils, 'getClientInstance').mockReturnValue({stop: stopSpy}); 
```
achieve the same thing without the extra class here and exported function in index?